common/common.go

```diff
@@ -31,6 +31,7 @@ const (
 	MetadataServiceIP      = "169.254.169.255"
 	MetadataServicePubPort = 80
 	MetadataServicePrvPort = 2375
+	MetadataServiceRegSock = "/var/run/rkt-metadata-svc.sock"
```

Comment:
I'm all for distros using /run to have only one tmpfs and having it up early. But I am unclear if the programs should migrate over to /run or continue using FHS paths and leave the fact that it'll all end up in /run as impl detail.

Don't have very strong feelings though. I actually used /run for flannel stuff (but I don't know if this was the right decision).